src/test/resources/Features/verifyEndpointsReturnCorrectResponse.feature

```diff
@@ -11,3 +12,11 @@ Feature: Verify endpoints return correct response
 
   Scenario: Verify cache_status endpoint returns status of "okay" for "all caches are populated"
     Then the cache_status endpoint returns status "OK" for all caches are populated status
+    
+  Scenario: Verify cache is re-initialized after an activity
+    Given I'm logged in as "ROLE_ADMIN"
+    And I navigate to ONC ACB Management page
+    And I access "ICSA Labs" ACB details
+    And I open the ACB edit form
+    When I edit ACB name and save    
```

Comment:
Reading this, it looks like, at the end of the scenario, the name has changed. I'd change this step to `I edit ACB name to be "ICSA Labs-edited" and save`, and add a step below that is `I edit ACB name to be "ICSA Labs" and save`.

That way it's clear that the name _is_ changing, but not permanently